test/nodetool/test_describering.py

```diff
@@ -0,0 +1,90 @@
+                    "127.0.0.1",
+                    "127.0.0.2"
+                ],
+                "endpoint_details": [
+                    {
+                        "host": "127.0.0.1",
+                        "datacenter": "datacenter1",
+                        "rack": "rack1"
+                    },
+                    {
+                        "host": "127.0.0.2",
+                        "datacenter": "datacenter2",
+                        "rack": "rack2"
+                    }
+                ]
+            },
+            {
+                "start_token": "9213626581013704850",
+                "end_token": "-9153143965931359657",
+                "endpoints": [
+                    "127.0.0.1"
+                ],
+                "rpc_endpoints": [
+                    "127.0.0.1"
+                ],
+                "endpoint_details": [
+                    {
+                        "host": "127.0.0.1",
+                        "datacenter": "datacenter1",
+                        "rack": "rack1"
+                    }
+                ]
+            }
+    ]
+    res = nodetool("describering", "ks", expected_requests=[
+        expected_request("GET", "/storage_service/schema_version", response=schema_version),
+        expected_request("GET", "/storage_service/describe_ring/ks", response=ring)])
+    assert res == f"""Schema Version:{schema_version}
+TokenRange: 
+\tTokenRange(start_token:-9153143965931359657, end_token:-9105705820509211664, endpoints:[127.0.0.1, 127.0.0.2], rpc_endpoints:[127.0.0.1, 127.0.0.2], endpoint_details:[EndpointDetails(host:127.0.0.1, datacenter:datacenter1, rack:rack1), EndpointDetails(host:127.0.0.2, datacenter:datacenter2, rack:rack2)])
+\tTokenRange(start_token:9213626581013704850, end_token:-9153143965931359657, endpoints:[127.0.0.1], rpc_endpoints:[127.0.0.1], endpoint_details:[EndpointDetails(host:127.0.0.1, datacenter:datacenter1, rack:rack1)])
+"""
+
+def test_describering_table(nodetool, scylla_only):
+    schema_version = "83541f12-c1bc-11ee-b55e-4d563ca8da4b"
+    ring = [
+            {
+                "start_token": "9213626581013704850",
+                "end_token": "-9153143965931359657",
+                "endpoints": [
+                    "127.0.0.1"
+                ],
+                "rpc_endpoints": [
+                    "127.0.0.1"
+                ],
+                "endpoint_details": [
+                    {
+                        "host": "127.0.0.1",
+                        "datacenter": "datacenter1",
+                        "rack": "rack1"
+                    }
+                ]
+            }
+    ]
+    res = nodetool("describering", "ks", "tbl", expected_requests=[
+        expected_request("GET", "/storage_service/schema_version", response=schema_version),
+        expected_request("GET", "/storage_service/describe_ring/ks", params={"table": "tbl"}, response=ring)])
+    assert res == f"""Schema Version:{schema_version}
```

Comment:
Do we merge adjacent tablets for describering? If not, this will fail if we start creating tables with >1 tablets.

Reply:
I don't see why? Again, here we just convert the JSON ScyllaDB sent us to text. No logic related to tablets here.

Reply:
I think it's the same confusion. A real scylladb might send multiple start_token/end_token tomorrow, but he mock one won't.